onnx/helper.py

```diff
@@ -1533,3 +1533,34 @@ def get_all_tensor_dtypes() -> KeysView[int]:
     :return: all tensor types from TensorProto
     """
     return mapping.TENSOR_TYPE_MAP.keys()
+
+
+attribute_proto_type_to_str = {
+    AttributeProto.FLOAT: "float",
+    AttributeProto.FLOATS: "floats",
+    AttributeProto.GRAPH: "graph",
+    AttributeProto.GRAPHS: "graphs",
+    AttributeProto.INT: "int",
+    AttributeProto.INTS: "ints",
+    AttributeProto.SPARSE_TENSOR: "sparse_tensor",
+    AttributeProto.SPARSE_TENSORS: "sparse_tensors",
+    AttributeProto.STRING: "string",
+    AttributeProto.STRINGS: "strings",
+    AttributeProto.TENSOR: "tensor",
+    AttributeProto.TENSORS: "tensors",
+    AttributeProto.TYPE_PROTO: "proto",
+    AttributeProto.TYPE_PROTOS: "protos",
+}
+
+
+def attr_type_to_str(attr_type: int) -> str:
```

Comment:
```suggestion
def _attr_type_to_str(attr_type: int) -> str:
```

1. I would make private
2. Consider not failing it but provide the original int value instead, so we always get _some_ error messages

Reply:
I'm kind of loathe to return the original int value as having mixed type returns seems undesirable. I could avoid raising an exception and return `"UNKNOWN"` or `"INVALID"` instead.

Reply:
I think "UNKNOWN" works. Thanks!